.buildkite/nightly-benchmarks/scripts/run-performance-benchmarks.sh

```diff
@@ -485,6 +485,7 @@ main() {
   # postprocess benchmarking results
   pip install tabulate pandas
   python3 $QUICK_BENCHMARK_ROOT/scripts/convert-results-json-to-markdown.py
+  python3 $QUICK_BENCHMARK_ROOT/scripts/compare-json-results.py -f $RESULTS_FOLDER/benchmark_results.json
```

Comment:
Perhaps we can add an ENV for exporting comparison reports, likes ```EXPORT_COMPARISON```. Looks like it is just needed when doing interactive benchmarking.